src/display/annotation_layer.js

```diff
@@ -96,14 +97,17 @@ AnnotationElementFactory.prototype =
  * @alias AnnotationElement
  */
 var AnnotationElement = (function AnnotationElementClosure() {
-  function AnnotationElement(parameters) {
+  function AnnotationElement(parameters, isRenderable) {
+    this.isRenderable = isRenderable;
```

Comment:
But `return null` in constructor doesn't make the `new`ed element become `null`.